extension/scripts/lib/tests/scroll.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function (){
```

Comment:
Can you put a comment saying that this is run on the client and what it tests for.